packages/gatsby-plugin-emotion/README.md

```diff
@@ -15,5 +15,10 @@ npm install --save gatsby-plugin-emotion emotion emotion-server react-emotion
 Add the plugin to your `gatsby-config.js`.
 
 ```js
-plugins: ["gatsby-plugin-emotion"];
+plugins: [
+    ["gatsby-plugin-emotion", options: {
+        // Accepts all options defined by `babel-plugin-emotion` plugin.
+        ... 
+    }]    
```

Comment:
I don't think this would work you need to declare it like that:
```js
plugins: [
  {
    resolve: `gatsby-plugin-emotion`,
    options: {
      // your options here
    }
  }
]
```

also if You are already updating it - can you format it so codeblock includes `module.exports = { }` like in https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-plugin-styled-components#how-to-use ?

Reply:
Good catch. Sorry about that.  I was referring to the .babelrc config instead of gatbys.